### PR TITLE
Add buildable crosses to throw degenerates on

### PR DESCRIPTION
**Pull request by rhicora**
- Adds handcrafting item "cross", which builds a cross for 10 wood
- Deconstructing crosses with a crowbar yields the wood back
- Add interactions to buckle and unbuckle mobs
- Add the cross sprite and sound effect

image

### CODE REVIEW

File: code/game/objects/structures/kitchen_spike.dm [hunk 1]
```diff
@@ -149,4 +149,84 @@
 	new /obj/item/stack/rods(loc, 4)
 	qdel(src)
 
+/obj/structure/kitchenspike/cross
+	name = "cross"
+	icon = 'icons/obj/cross.dmi'
+	icon_state = "cross"
+	desc = "Degenerates like you belong on one of these."
+	density = 1
+	anchored = 1
```
Comment (LetterN) [inline code/game/objects/structures/kitchen_spike.dm]: TRUE

File: code/game/objects/structures/kitchen_spike.dm [hunk 2]
```diff
@@ -149,4 +149,84 @@
...
+	buckle_lying = 0
+	can_buckle = 1
```
Comment (LetterN) [inline code/game/objects/structures/kitchen_spike.dm]: TRUE

File: code/game/objects/structures/kitchen_spike.dm [hunk 3]
```diff
@@ -149,4 +149,84 @@
...
+	bound_height = 64
+
+/obj/structure/kitchenspike/cross/attackby(obj/item/I, mob/user, params)
+	if(istype(I, /obj/item/crowbar))
+		if(!src.buckled_mobs)
+			playsound(loc, 'sound/items/Crowbar.ogg', 100, 1)
+			if(do_after(user, 30/I.toolspeed, target = src))
+				user << "<span class='notice'>You tear down the cross.</span>"
```
Comment (LetterN) [inline code/game/objects/structures/kitchen_spike.dm]: to_chat()

File: code/game/objects/structures/kitchen_spike.dm [hunk 4]
```diff
@@ -149,4 +149,84 @@
...
+				new /obj/item/stack/sheet/mineral/wood(loc, 10)
+				qdel(src)
+		else
+			user << "<span class='notice'>You can't do that while something's on the cross!</span>"
```
Comment (LetterN) [inline code/game/objects/structures/kitchen_spike.dm]: to_chat()

File: code/game/objects/structures/kitchen_spike.dm [hunk 5]
```diff
@@ -149,4 +149,84 @@
...
+			L.loc = src.loc
+			L.emote("scream")
+			L.adjustBruteLoss(10)
+			L.buckled = src
+			L.dir = 2
+			buckle_mob(L, force=1)
+			L.pixel_y = 26
+			L.overlays += image('icons/obj/cross.dmi', "lashing")
+			return
+		user << "<span class='danger'>You can't use that on the cross!</span>"
```
Comment (LetterN) [inline code/game/objects/structures/kitchen_spike.dm]: to_chat()

File: code/game/objects/structures/kitchen_spike.dm [hunk 6]
```diff
@@ -149,4 +149,84 @@
...
+
+		else
+			M.visible_message(\
+			"<span class='warning'>[M.name] struggles to break free from the [src]!</span>",\
+			"<span class='notice'>You struggle to break free from the [src], exacerbating your wounds! (Stay still for two minutes.)</span>",\
+			"<span class='italics'>You hear violent scraping and struggling.</span>")
+			M.adjustBruteLoss(20)
+			if(!do_after(M, 1200, target = src))
+				if(M && M.buckled)
+					M << "<span class='warning'>You fail to free yourself!</span>"
```
Comment (LetterN) [inline code/game/objects/structures/kitchen_spike.dm]: to_chat()

File: code/game/objects/structures/kitchen_spike.dm [hunk 7]
```diff
@@ -149,4 +149,84 @@
...
+				return
+		if(!M.buckled)
+			return
+		M.pixel_y = 0
+		M.adjustBruteLoss(40)
+		src.visible_message(text("<span class='danger'>[M] falls free of the [src]!</span>"))
```
Comment (LetterN) [inline code/game/objects/structures/kitchen_spike.dm]: to_chat()

**Comment by AndrewL97** [comment]
These changes seem to be contained in #177

**Comment by rhicora** [comment]
Closing this, it's covered in Casino Update

**Pull request closed without merging**